client_auto.go

```diff
@@ -0,0 +1,95 @@
+package hdfs
+
+import (
+	"encoding/xml"
+	"errors"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+)
+
+type Property struct {
```

Comment:
This and `Result` don't need to be capitalized, since they aren't exported. Also I would name `Result` something like `propertyList`.